### PR TITLE
547 release notes

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**
May need to add a couple more things to the docs.

### CODE REVIEW

**Review by MichaelBaj — APPROVED** [review]

File: docs/release_notes_128t_5.4.md [hunk 1]
```diff
@@ -24,6 +24,54 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-46562 Allow targeting another router or node when saving tech-support-info:** GUI: A button has been added to the **Logs** page in the GUI to download a tech-support-info bundle. This allows downloading a router's `tech-support-info` directly from the Conductor GUI. <br />
+PCLI: The PCLI command `save tech-support-info` can now collect logs from another node. Using the Conductor's PCLI, a `tech-support-info` bundle can be collected from a Managed Router or the HA peer.
+------
+- **I95-46574 Security History Export:** An Export button has been to the Security History page.
+------
+- **I95-46747 Improved the Password user experience:** You now are re-propmpted up to three times for the current password if it is incorrect. If a new password does not meet the strength check, you are prompted with that information, and required to update the password.
```
Comment (Bhautik) [inline docs/release_notes_128t_5.4.md:42]: Seems like a spelling mistake at re-prompted

**Review by Bhautik — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_5.4.md [hunk 2]
```diff
@@ -24,6 +24,54 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-40333 Save credentials for accessing SSR software repositories:** `set software access-token` is a new PCLI command to save credentials for accessing SSR software repositories. This provides a way to run `install128t repo authenticate` without dropping to a linux shell. For additional information on this command, see [`set software access-token`](cli_reference.md#set-software-access-token).
+------
+- **I95-46562 Allow targeting another router or node when saving tech-support-info:** GUI: A button has been added to the **Logs** page in the GUI to download a tech-support-info bundle. This allows downloading a router's `tech-support-info` directly from the Conductor GUI. <br />
+PCLI: The PCLI command `save tech-support-info` can now collect logs from another node. Using the Conductor's PCLI, a `tech-support-info` bundle can be collected from a Managed Router or the HA peer.
+------
+- **I95-46574 Security History Export:** An Export button has been to the Security History page.
```
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.4.md:40]: done

File: docs/release_notes_128t_5.4.md (hunk 1 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.4.md:42]: fixed

File: docs/intro_downloading_iso.md [hunk 3]
```diff
@@ -29,6 +29,10 @@ With releases 4.5.6-1 and 5.0.0-1 v2 (and greater), we now provide a single ISO
 - 128T-4.5.6-1.el7.OTP.v1.x86_64.iso
 - 128T-5.0.0-1.el7.OTP.v2.x86_64.iso
 
+With release 5.4.7-9 (and greater) a single ISO is provided that supports Installation workflows.
```
Comment (Bhautik) [inline docs/intro_downloading_iso.md:32]: it needs to be 5.4.7-7

File: docs/intro_downloading_iso.md [hunk 4]
```diff
@@ -29,6 +29,10 @@ With releases 4.5.6-1 and 5.0.0-1 v2 (and greater), we now provide a single ISO
 - 128T-4.5.6-1.el7.OTP.v1.x86_64.iso
 - 128T-5.0.0-1.el7.OTP.v2.x86_64.iso
 
+With release 5.4.7-9 (and greater) a single ISO is provided that supports Installation workflows. 
+
+- 128T-5.4.7-5.el7.ISO.v1.x86_64.iso
```
Comment (Bhautik) [inline docs/intro_downloading_iso.md:34]: it needs to be 5.4.7-7

**Review by Bhautik — CHANGES_REQUESTED** [review]

**Review by Bhautik — APPROVED** [review]

**Review by jeffmart-jnpr — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**